File: app/src/main/java/com/magenta/rx/rxa/model/record/Definition.java
package com.magenta.rx.rxa.model.record;

public class Definition implements Texted {

    private String text;
    private String pos;
    private String ts;
    private Transcription[] tr;

    public Definition(String text, String pos, String ts, Transcription[] tr) {
        this.text = text;
        this.pos = pos;
        this.ts = ts;
        this.tr = tr;
    }

    public Definition() {
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getPos() {
        return pos;
    }

    public void setPos(String pos) {
        this.pos = pos;
    }

    public String getTs() {
        return ts;
    }

    public void setTs(String ts) {
        this.ts = ts;
    }

    public Transcription[] getTr() {
        return tr;
    }

    public void setTr(Transcription[] tr) {
        this.tr = tr;
    }
}

File: app/src/main/kotlin/com/magenta/rx/kotlin/record/Definition.kt
package com.magenta.rx.kotlin.record

class Definition(private var text: String?, private var pos: String?, private var ts: String?, private var tr: Array<Transcription>?) {

    fun getText() = text

    fun getPos() = pos

    fun getTs() = ts

    fun getTr() = tr

    fun setText(text: String) {
        this.text = text
    }

    fun setPos(pos: String) {
        this.pos = pos
    }

    fun setTs(ts: String) {
        this.ts = ts
    }

    fun setTr(tr: Array<Transcription>) {
        this.tr = tr
    }
}